### ISSUE TITLE
operator-no-unspaced: "Expected single space before", but there is a single space

### ISSUE THREAD

**Issue by jens-duttke** [kristerkari/stylelint-scss#440]
The "operator-no-unspaced" (set to `true`) returns the error:

> Expected single space before "\*"

for the following code:

```scss
.container {
	// IE7-hack
	@at-root * + html & {
		width: 176px;
		height: 176px;
	}
}
```
but like you see, there is a single space before `*`.

For this code:

```scss
.container {
	@at-root * {
		color: red;
	}
}
```
it even returns:

> Expected single space before "\*"
>  Expected single space after "\*"

but there is a single space before and after the `*`.

I'm using
```
"stylelint": "13.0.0"
"stylelint-scss": "3.13.0"
```

**Comment by kristerkari** [comment]
Thanks! I'll have a look at it.

**Comment by kristerkari** [comment]
Fixed in v3.14.2
https://github.com/kristerkari/stylelint-scss/releases/tag/3.14.2
